package.json

```diff
@@ -47,7 +47,7 @@
     "react-dom": "^16.8.6",
     "react-helmet": "^5.2.0",
     "react-measure": "^2.1.2",
-    "react-router-dom": "^5.0.1",
+    "react-router-dom": "^4.3.1",
```

Comment:
The `peerDependency` version of `react-router-dom` in `ot-ui` has been updated so this downgrade of the version should no longer be done.

Reply:
Addressed